### DISCUSSION TITLE
Problems with DigitalOcean and ETA

### DISCUSSION THREAD

**Discussion by ghost** [eta-dev/eta#88] [Q&A]
Hello,
I've contacted the support of DigitalOcean explaining that their App service causes problems with Eta but it doesn't cause any problems when I deploy it on a droplet by myself.

I've told them and they asked me if the ETA templating engine accesses the /proc/cpuinfo of the system?
Here a quote:
```
Does that engine need access to /proc/cpuinfo to pull that data?
App Platform is a much more isolated environment and wouldn't have CPU access information within a container. 
```

**Comment by shadowtime2000** [comment]
@damiantoczek Eta shouldn't be pulling data from `/proc/cpuinfo`. We only load stuff from the system when `renderFile` is used and/or `includeFile` and other fs related actions.

**Reply by shadowtime2000** [reply]
Could be a problem with Express or whatever you are using Eta with.

**Reply by ghost** [reply]
> Could be a problem with Express or whatever you are using Eta with.

Hi, could we talk via DM?

**Reply by shadowtime2000** [reply]
@damiantoczek Sorry for replying so late, sure! If you join the Gitter room you will be able to DM me or if you are in a Discord server I am in you can DM me.
